### PR TITLE
[full ci] Refactor robot utils for OOB power off/on, fix test

### PR DESCRIPTION
**Pull request by andrewtchin**

### CODE REVIEW

**Comment by mhagen-vmware** [comment]
minor comment

lgtm

File: tests/resources/Util.robot [hunk 1]
```diff
@@ -439,9 +439,27 @@ Install Harbor To Test Server
...
+Power Off VM OOB
+    [Arguments]  ${vm}
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc vm.power -off ${vch-name}/"${vm}"
+    Run Keyword If  '%{HOST_TYPE}' == 'VC'  Should Be Equal As Integers  ${rc}  0
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run And Return Rc And Output  govc vm.power -off "${vm}"
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Be Equal As Integers  ${rc}  0
+    Log To Console  Waiting for VM to power off ...
+    Wait Until VM Powers Off  "${vm}"
+
+Power On Container OOB
```
Comment (andrewtchin) [inline tests/resources/Util.robot]: good catch

**Comment by jzt** [comment]
LGTM

**Comment by andrewtchin** [comment]
♻️

**Pull request merged**